smile_id_core/api_base.py

```diff
@@ -11,12 +11,13 @@ class ApiBase:
     SERVER_URL: str
     """This can be one of `constants.Servers` entries"""
 
-    def __init__(self, partner_id: str, api_key: str):
+    def __init__(self, partner_id: str, api_key: str, api_version: int = 1):
```

Comment:
I believe it is not necessary to pass an argument to the constructor - rather, define it as a class attribute. This would eliminate the need to use 2 `super().__init__` calls in the subclasses.
But all in all, this new class-based approach looks a lot more sensible :)

Reply:
Thank you so much. Sure you are right. Let me do it.